clang/test/CodeGen/systemz-charset.c

```diff
@@ -0,0 +1,35 @@
+// RUN: %clang_cc1 %s -emit-llvm -triple s390x-none-zos -fexec-charset IBM-1047 -o - | FileCheck %s
```

Comment:
Should there be tests here for predefined variables like `__func__` and `__PRETTY_FUNCTION__`, for predefined macros like `__FILE__`, and for stringization of identifiers and strings?